src/compute-client/src/protocol/command.rs

```diff
@@ -431,6 +434,9 @@ impl ComputeParameters {
             self.enable_operator_hydration_status_logging =
                 enable_operator_hydration_status_logging;
         }
+        if enable_lgalloc_eager_reclamation.is_some() {
+            self.enable_lgalloc_eager_reclamation = enable_lgalloc_eager_reclamation;
```

Comment:
Seems untested

Reply:
Tested manually, it's a kill-switch if it would cause problems.